src/index.html

```diff
@@ -7,6 +7,99 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div 
+        class="calendar__cell"></div>
```

Comment:
Combine to one line. Check all other cases.

Reply:
i'm fix, please check my work)